packages/waku/src/lib/builder/build.ts

```diff
@@ -64,6 +64,7 @@ import { deployCloudflarePlugin } from '../plugins/vite-plugin-deploy-cloudflare
 import { deployDenoPlugin } from '../plugins/vite-plugin-deploy-deno.js';
 import { deployPartykitPlugin } from '../plugins/vite-plugin-deploy-partykit.js';
 import { deployAwsLambdaPlugin } from '../plugins/vite-plugin-deploy-aws-lambda.js';
+import { appendFileSync } from 'node:fs';
```

Comment:
The other vite deploy operations are sync which is why this is not using a promise.